### PR TITLE
Bugfix/fix object refresh after upload

### LINKED ISSUE #110: No list refresh after upload
When uploading a file to an bucket (Firefox + Windows), it gives a message that transfer was OK, but **the file does not appear unless one refreshes the folder from the browser.**  It does not even help to go to another folder meanwhile.

In my opinion this is a showstopper to release this for public use.

### PR DESCRIPTION
**Pull request by sampsapenna**
### Description

Fixes issue #110 – the objects in a container are now dynamically refreshed whenever an object upload completes. The refreshes are done upon completion of each uploaded file. File refreshes work also inside pseudofolders.
Additionally the PR fixes a newfound issue in the upload button, which did not re-register to the UI correctly after being replaced with upload cancellation, and implements uploads into pseudofolders from the button, and from drag-n-drop.

### Related issues

* Fixes #110 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (this needs a follow up PR)

### Changes Made

* Move object listing to Vuex to make allow updating objects outside `ObjectTable` component
* Remove caching object listings since checking for new content requires a full `GET` with current backend implementation, making caching somewhat unnecessary
* Move upload button from `FolderUploadForm` into its own component, to enable re-registration to `Resumable` object upon reactivation in the UI when an upload completes
* Rewrite `getUploadUrl` in `main.js` to enable uploading single files into pseudofolders (+ folders with drag-n-drop)
* Removed some unnecessary stuff

### Testing

- [ ] Unit Tests
- [ ] Integration Tests
- [x] Tests do not apply
- [ ] Needs testing (start an issue or do a follow up PR about it)

### CODE REVIEW

File: swift_browser_ui_frontend/src/common/store.js [hunk 1]
```diff
@@ -2,8 +2,12 @@
 import Vue from "vue";
 import Vuex from "vuex";
 
-import { recursivePruneCache } from "@/common/conv";
+// import { recursivePruneCache } from "@/common/conv";
```
Comment (blankdots) [inline swift_browser_ui_frontend/src/common/store.js:5]: is this a forgotten comment?

File: swift_browser_ui_frontend/src/common/store.js [hunk 2]
```diff
@@ -43,16 +47,50 @@ const store = new Vuex.Store({
...
+        ).catch(() => {
+          state.isLoading = false;
+        });
+      }
+      else {
+        getObjects(container).then((ret) => {
+          if (ret.status != 200) {
+            state.isLoading = false;
+          }
+          // state.objectCache = recursivePruneCache(state.objectCache);
```
Comment (blankdots) [inline swift_browser_ui_frontend/src/common/store.js:84]: same here ?

File: swift_browser_ui_frontend/src/common/store.js (hunk 1 above)
Comment (sampsapenna) [reply swift_browser_ui_frontend/src/common/store.js:5]: Yup, from when I was thinking if I should re-implement the caching anyway

I'll remove those

Comment (sampsapenna) [reply swift_browser_ui_frontend/src/common/store.js:5]: I think that this will make `recursivePruneCache` unused, so that should probably go as well

Comment (sampsapenna) [reply swift_browser_ui_frontend/src/common/store.js]: Removed 👍🏼

File: swift_browser_ui_frontend/src/common/store.js (hunk 2 above)
Comment (sampsapenna) [reply swift_browser_ui_frontend/src/common/store.js]: Yup, removed 👍🏼

**Review by blankdots — APPROVED** [review]

**Pull request merged**